packages/editor/src/bindings/post-meta.js

```diff
@@ -9,26 +9,65 @@ import { store as coreDataStore } from '@wordpress/core-data';
 import { store as editorStore } from '../store';
 import { unlock } from '../lock-unlock';
 
-function getMetadata( registry, context, registeredFields ) {
-	let metaFields = {};
-	const type = registry.select( editorStore ).getCurrentPostType();
+/**
+ * Get a list of post meta fields with their values and labels
```

Comment:
Nit: the documentation by convention should be written in 3rd person:
```suggestion
 * Gets a list of post meta fields with their values and labels
```

[Source](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/):

> Summaries should be clear, simple, and brief. Avoid describing “why” an element exists, rather, focus on documenting “what” and “when” it does something.
> A function, hook, class, or method is a third-person singular element, meaning that third-person singular verbs should be used to describe what each does.